ci/vulcan/home.nix

```
@@ -0,0 +1,37 @@
+{ config, pkgs, ... }:
+
+# Note: This file must be symlinked to the proper home-manager config directory.
+
+let
+  webring-flake = (builtins.getFlake "path:/home/ring/webring").packages.x86_64-linux;
+  webring = webring-flake.phwebring;
+  default-static = webring-flake.default-static;
+in {
+  home.username = "ring";
+  home.homeDirectory = "/home/ring";
+  home.stateVersion = "24.11";
+
+  home.packages = [ webring ];
+
+  home.file."webring-data/static".source = default-static;
+
+  # This will automatically get restarted when rebuilding the home directory
+  systemd.user.services.phwebring = {
+    Unit = {
+      Description = "Purdue Hackers webring";
+    };
```

```suggestion
    Unit.Description = "Purdue Hackers webring";
```

Nit: You can condense nested attribute sets like this, if you want. This is down to stylistic choice, however.